dts/msm8916/msm8916-samsung-r00.dts

```diff
@@ -15,7 +15,16 @@
 		compatible = "samsung,gt58lte-aus", "qcom,msm8916", "lk2nd,device";
 		lk2nd,match-bootloader = "T355Y*";
 	};
+        on7lte-kor {
```

Comment:
We use tabs not spaces for indentation here.

```suggestion
	};

	on7lte-kor {
```

Reply:
Also here should probably be empty line (you added spaces in this file in previous separate commit and then add new content without blank line?)